provider/tests/test_mgmt_cmd.py

```diff
@@ -23,26 +28,32 @@ def setUp(self):
         super(_TokenDeletionTestCase, self).setUp()
         self.command = Command()
 
+        # Make User and Client.
+        user = User.objects.create_user(uuid.uuid4().hex, '%s@example.com' % uuid.uuid4().hex)
```

Comment:
A tiny thing: it'd be more straightforward for this test to use the same UUID for both the username and the first part of the email address - instead of using two separate UUIDs.

Reply:
updated